docs/guides/add-validator-with-curl.md

```diff
@@ -54,6 +55,22 @@ Pre-sale participants have already been designated as Consortium Members on the
 
 :::
 
+:::info UNBONDED CAM TOKENS
+
+In the Camino Network, CAM tokens can be utilized as securities for various purposes such as deposit rewards, validator stakes, or DAO proposals. There are two types of these securities: **deposits** and **bonds**.
+
+Individuals can deposit CAM tokens for a certain period of time to earn rewards. In this scenario, the tokens are "**deposited**" and their state is "**locked**".
```

Comment:
What does it mean exactly, that their state is **locked**?

Reply:
Just trying to say that these tokens would be locked. Remote "state" part, so it's not confusing.